core/shared/src/main/scala/scalaz/zio/ZIO.scala

```diff
@@ -1501,6 +1514,11 @@ trait ZIOFunctions extends Serializable {
    * Returns information about the current fiber, such as its fiber identity.
    */
   final def descriptor: UIO[Fiber.Descriptor] = ZIO.Descriptor
+
+  /**
+   * Provides access to the list of fibers forked by this fiber.
+   */
+  final def supervising: UIO[Iterable[Fiber[_, _]]] = ZIO.Supervising
```

Comment:
This is nitpicking: I'm looking at `descriptor` just above and wonder if this one shouldn't just be called `fibers` instead. What do you think?

Reply:
Yes, I like that! The ambiguity bothered me as well.

Reply:
Or how about `children` or `childFibers`?